coderd/organizations.go

```diff
@@ -68,10 +68,11 @@ func (api *API) postOrganizations(rw http.ResponseWriter, r *http.Request) {
 	var organization database.Organization
 	err = api.Database.InTx(func(tx database.Store) error {
 		organization, err = tx.InsertOrganization(ctx, database.InsertOrganizationParams{
-			ID:        uuid.New(),
-			Name:      req.Name,
-			CreatedAt: dbtime.Now(),
-			UpdatedAt: dbtime.Now(),
+			ID:          uuid.New(),
+			Name:        req.Name,
+			CreatedAt:   dbtime.Now(),
+			UpdatedAt:   dbtime.Now(),
+			Description: "",
```

Comment:
It looks like something we could expose via HTTP request.

Reply:
Maybe we should remove it? Of course, could be a separate PR.

Reply:
Just the field, or the concept entirely? The field itself is probably just a forklift from v1. I feel that the entire concept also is, but that's a much wider topic.

Reply:
Yes, the field, but keep in mind that maybe I just don't see the full picture.

Reply:
However, some [isuses](https://github.com/coder/coder/issues/8870) do mention exposing organizations. So maybe best to leave the field as-is for now.